src/parser/content-evaluator-module.ts

```diff
@@ -109,7 +109,11 @@ export class ContentEvaluatorModule extends BaseModule {
     );
 
     if (Object.keys(relevancesByAi).length !== commentsToEvaluate.length + prCommentsToEvaluate.length) {
-      throw this.context.logger.fatal("Relevance / Comment length mismatch!", { relevancesByAi, commentsToEvaluate });
+      throw this.context.logger.fatal("Relevance / Comment length mismatch!", {
```

Comment:
```suggestion
      throw this.context.logger.error("Relevance / Comment length mismatch!", {
```

Reply:
We are intending to link an emergency triage telegram channel to fatal errors so it should only be 1. consistent errors and 2. probably kernel related